app/jobs/send_user_invitation_job.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class SendUserInvitationJob < ActiveJob::Base
+  queue_as :user_invites
```

Comment:
A new queue just for this no plz 😭 for me either `low` or `default`

Reply:
OK I don't know 🤷‍♂️ .`default` then?